Formula/lxc.rb

```diff
@@ -1,8 +1,8 @@
 class Lxc < Formula
   desc "CLI client for interacting with LXD"
   homepage "https://linuxcontainers.org"
-  url "https://linuxcontainers.org/downloads/lxd/lxd-5.4.tar.gz"
-  sha256 "e234b67c5077d05e22f958e3256bd91f262450545993d0aaf1b4ce2bdb99393a"
+  url "https://linuxcontainers.org/downloads/lxd/lxd-5.5.tar.gz"
```

Comment:
not accessible
```
$ curl -I https://linuxcontainers.org/downloads/lxd/lxd-5.5.tar.gz
HTTP/1.1 404 Not Found
server: nginx/1.18.0 (Ubuntu)
date: Fri, 19 Aug 2022 01:39:55 GMT
content-type: text/html
content-length: 564
```

Reply:
Did a force push now to retry as the file is available now.

Reply:
That makes sense. 😄